apps/src/templates/lessonOverview/utils.js

```diff
@@ -0,0 +1,68 @@
+const GDOCS_HOST = 'docs.google.com';
```

Comment:
nit: maybe this file should be called something like `googleDocsUtils.js` or `googleDriveUtils.js`?